patches/kaggle_web_client.py

```diff
@@ -12,7 +12,7 @@
                             CredentialError, BackendError, ValidationError)
 
 class KaggleWebClient:
-    TIMEOUT_SECS = 90
+    TIMEOUT_SECS = 600
```

Comment:
Have you considered using an env var instead of hard-coding this into the build?

Reply:
I guess I haven't. How hard would it be to add and adjust an env var? 

I don't expect this to be something that we change regularly. Once we have an operation-based API it will likely go back to a more standard 60-90 second timeout, but that would be a one-time change.

Reply:
If it's an env var, then you can pass it in when the kernel is created (you should have a default in the container if the env var is not set). And it won't care what docker version you're using.

But if you hard code it, then only newer kernels that use a newer docker container will use updated timeouts.

Reply:
@rosbo Do you have an opinion here or should we just LGTM and let this through?

Reply:
> Once we have an operation-based API it will likely go back to a more standard 60-90 second timeout

Will you need another change to the Docker image to use the operation based API? If so, then, it might be best to hardcode the timeout value for the api you are calling.

Reply:
We also have the option of patching over these source files on the docker image before booting the container.
We actually already do that for one file.

Reply:
To clarify,

If with this change we are calling API A (not operation-based) and requires a timeout of 600s.
And then later we have a change calling API B (operation-based) and only needs a timeout of 90s.

It might be better to hardcode the timeout in the image so it is in sync with the API being called.

Reply:
Yes, this is currently calling API A and the replacement will be a call to API B with a shorter timeout. The changes to switch to the new API will be more substantial and will involve another change to the Docker image.

Reply:
Approved. I still wonder if we'd be better served to have the entire patches/ directory added at startup instead of committed to Docker, but I guess we can let this go through before talking that over.